MediaBrowser.Common/Json/JsonDefaults.cs

```diff
@@ -0,0 +1,30 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using MediaBrowser.Common.Json.Converters;
+
+namespace MediaBrowser.Common.Json
+{
+    /// <summary>
+    /// Helper class for having compatible JSON throughout the codebase.
+    /// </summary>
+    public static class JsonDefaults
+    {
+        /// <summary>
+        /// Gets the default <see cref="JsonSerializerOptions" /> options.
+        /// </summary>
+        /// <returns>The default <see cref="JsonSerializerOptions" /> options.</returns>
+        public static JsonSerializerOptions GetOptions()
+        {
+            var options = new JsonSerializerOptions()
+            {
+                ReadCommentHandling = JsonCommentHandling.Disallow,
```

Comment:
eh, why?

Reply:
Json doesn’t support comments by default, and who would have json comments in their db?

Reply:
no idea, was just curios :)